Tasks/CondaEnvironment/conda.ts

```diff
@@ -0,0 +1,44 @@
+import * as fs from 'fs';
+import * as path from 'path';
+
+import * as task from 'vsts-task-lib/task';
+
+import * as internal from './conda_internal';
+import { Platform } from './taskutil';
+
+interface TaskParameters {
+    environmentName: string,
+    packageSpecs?: string
+    otherOptions?: string,
+    installConda?: boolean
+}
+
+export async function condaEnvironment(parameters: Readonly<TaskParameters>, platform: Platform): Promise<void> {
```

Comment:
nit: I feel like this could have a better name.  `condaEnvironment` is a function without a verb in the name: `useCondaEnvironment` or the like would be an improvement, IMO.

Reply:
I named the function for the task.  The pattern here is that `main.ts` communicates with the task runner, and this function contains the actual implementation for the task.

To the point, perhaps the name of the task could have a verb in it.  We could say "Activate Conda Environment," but that makes it sound like the environment has to exist already.

Reply:
I feel that code should read like code.  To me, that's more important than this matching a design pattern.  `Conda Environment` may make sense as a name for a task, but it doesn't make sense as a function.  If you feel that this pattern is important enough to change the name of the task, I'd support that decision; I don't think that the name of the task should dictate the name of the function in the code if makes the code read weird.

Reply:
I'm not sure I follow your example.  I would say `getFather`.  Caching would be an implementation detail.  `father` *is* a verb, and I would expect it would get the child of `Lucas Killgore`, which in this case would be `null`, or `empty`, or possibly even `stuffed panda bears` depending on definitions.  From your comment, I would expect that you would think it should return `Darth Vader`.  In any case, leaving 'get' off makes the call ambiguous.

FWIW, this would be fine to me: `const environment = new CondaEnvironment(); environment.set();`.

I think it comes down to me disagreeing with the central conceit: 'if functions are your main unit of code organization, not all of your functions are verbs'.  No, I think they should be, or at least, this is not an exception to that case.